raster.py

```diff
@@ -70,13 +69,19 @@ def get_animations_path(filename: str) -> str:
   return f"{root}{ANIMATIONS_BASE}{filename}"
 
 def mount_gdrive():
-  global MOUNTED
-  if not MOUNTED:
-    MOUNTED = True
+  if not os.path.exists(GDRIVE_BASE):
     # Access data stored on Google Drive
     if GDRIVE_BASE:
+      try:
         from google.colab import drive
         drive.mount(GDRIVE_BASE)
+      except ModuleNotFoundError:
+        print('WARNING, GDRIVE NOT MOUNTED! USING LOCAL FS!!!')
+
+      if not os.path.exists(GDRIVE_BASE):
```

Comment:
Unless I'm missing something we should test this case.

Reply:
it is tested in raster_test by triggering an error during the run thru